Marlin/src/module/stepper_indirection.cpp

```diff
@@ -557,14 +568,15 @@
   void tmc_init(TMCMarlin<TMC2660Stepper, AXIS_LETTER, DRIVER_ID, AXIS_ID> &st, const uint16_t mA, const uint16_t microsteps, const uint32_t, const bool) {
     st.begin();
 
-    TMC2660_n::CHOPCONF_t chopconf{0};
-    chopconf.tbl = 1;
-    chopconf.toff = chopper_timing.toff;
-    chopconf.hend = chopper_timing.hend + 3;
-    chopconf.hstrt = chopper_timing.hstrt - 1;
-    #if ENABLED(SQUARE_WAVE_STEPPING)
-      chopconf.dedge = true;
-    #endif
+    TMC2660_n::CHOPCONF_t chopconf {
+      .toff     = chopper_timing.toff,
+      .hstrt    = chopper_timing.hstrt - 1,
+      .hend     = chopper_timing.hend + 3,
+      .tbl      = 1,
+      #if ENABLED(SQUARE_WAVE_STEPPING)
+        .dedge  = true // Not in TMCStepper v0.4.1 yet??
```

Comment:
@teemuatlut — It looks like the `dedge` field is not in the `CHOPCONF_t` structure in v0.4.1. Can you confirm?